Source/controls/plrctrls.cpp

```diff
@@ -1171,6 +1185,17 @@ void StashMove(AxisDirection dir)
 		Point mousePos = GetStashSlotCoord(ActiveStashSlot);
 		// Stash coordinates are all the top left of the cell, so we need to shift the mouse to the center of the held item
 		// or the center of the cell if we have a hand cursor (itemSize will be 1x1 here so we can use the same calculation)
+		if (holdItem.isEmpty()) {
+			const StashStruct::StashCell itemIdAtActiveStashSlot = Stash.GetItemIdAtPosition(ActiveStashSlot);
+			if (itemIdAtActiveStashSlot != StashStruct::EmptyCell) {
+				const Item stashItem = Stash.stashList[itemIdAtActiveStashSlot];
+				const Point firstSlotOnItem = FindFirstStashSlotOnItem(itemIdAtActiveStashSlot);
+				itemSize = GetInventorySize(stashItem);
+				mousePos = GetStashSlotCoord(firstSlotOnItem);
+				ActiveStashSlot = FindClosestStashSlot(mousePos);

```

Comment:
This line is actually unnecessary, it causes a behaviour difference compared to moving through the inventory. If you comment it out the gamepad navigation "remembers" the last column/row that was active when moving along staggered items. As-is you end up laddering to the top left cell of the stash.

https://github.com/user-attachments/assets/4888adea-d590-414b-aab9-134e1ec24ecc



https://github.com/user-attachments/assets/ceef7785-8e35-474b-b8ba-3086e1aad768

Reply:
removed